### DISCUSSION TITLE
Thoughts about the new swipe gesture behaviour

### DISCUSSION THREAD

**Discussion by Arbiter313** [notaapp/talk#368] [Feature Requests]
I've been using the most recent release of Nota for the past few weeks and had a few thoughts and questions about the new swipe gesture behaviour.  

I have found the behaviour of the swipe gesture to be unreliable and confusing. Sometimes it switches the note in a particular tab, and sometimes it switches to another tab completely. Clarification on the exact behaviour would be much appreciated. I can only assume the swipe behaviour was changed so it is more similar to browsers (swipe to 'go back'), but would be interested to get the developer's thoughts on why it was changed. 

Personally, I miss the behaviour of the old gesture. There is something about the physical gesture on the trackpad translating to a manipulation of the digital Nota canvas that was extremely satisfying. I believe something similar like swiping back and forward to switch between consecutive tabs would also be a satisfying behaviour. Again, I would be interested to hear others thoughts on this.

**Comment by erusev** [comment]
I understand. I'd guess that what makes it a bit confusing is that unlike the browser, where each tab is a separate history thread, here, we have a single thread and as a result, going back and forward sometimes changes tabs.

We also miss being able to swipe to toggle the sidebar, but at the same time, using swipe to go back and forward also seems very useful. Figuring out the right balance for things like this is tricky, and we appreciate the feedback.

**Reply by Arbiter313** [reply]
I don't envy you in finding the right balance; it must be extremely difficult! Perhaps you could try swapping the current behaviours for 'swipe' and 'cmd + swipe' and see how that feels. Or even have a setting that allows the user to set swipe behaviour (although I understand the developer's desire to streamline the settings as much as possible). Again, I don't envy you.
